src/unix/bsd/freebsdlike/dragonfly/mod.rs

```diff
@@ -1522,6 +1522,18 @@ pub const NGROUPS: usize = 16;
 pub const RB_PAUSE: ::c_int = 0x40000;
 pub const RB_VIDEO: ::c_int = 0x20000000;
 
+// net/route.h
+pub const RTF_CLONING: ::c_int = 0x100;
+pub const RTF_PRCLONING: ::c_int = 0x10000;
+pub const RTF_WASCLONED: ::c_int = 0x20000;
+pub const RTF_MPLSOPS: ::c_int = 0x1000000;
+
+pub const RTM_VERSION: ::c_int = 7;
```

Comment:
shall I remove this place holder?

Reply:
Would be inconsistent with macOS:
https://github.com/rust-lang/libc/blob/acc7bb1a38e2b9d6a18f28279d80d464c4c25089/src/unix/bsd/apple/mod.rs#L4770

Besides, RTM_VERSION is [used by iroh-net](https://github.com/n0-computer/iroh/blob/370075c6d5689ad4349664eb9b51ec0a5e7e4e81/iroh-net/src/net/interfaces/bsd.rs#L371).